.eslintrc

```diff
@@ -14,5 +14,16 @@
     "jsdoc": {
       "mode": "typescript"
     }
-  }
+  },
+  "overrides": [
+    {
+     "files": [
+       "**/test/*.js",
+       "**/*.test.js"
+     ],
+     "rules": {
+       "import/no-unresolved": "off"
```

Comment:
An attempt to prevent this eslint error:


But I couldn't get `overrides.files` to resolve right. Like a [previous attempt](https://github.com/wp-plugin-sidekick/wpps-scripts/pull/10#discussion_r839027701), I couldn't get them to resolve to `fse-studio`. They're supposed to be relative to the path of this `.eslintrc` file.